internal/pkg/api/server/templates/service/service.go

```diff
@@ -314,19 +319,51 @@ func (s *service) UpgradeInstanceValidateInputs(dependencies.Container, *Upgrade
 	return nil, NotImplementedError{}
 }
 
-func repositories() []model.TemplateRepository {
-	return []model.TemplateRepository{
-		{
-			Type: "git",
-			Name: repository.DefaultTemplateRepositoryName,
-			Url:  repository.DefaultTemplateRepositoryUrl,
-			Ref:  "api-demo",
-		},
-	}
+func repositories(repoPath string) ([]model.TemplateRepository, error) {
+	if repoPath == "" {
+		return []model.TemplateRepository{
+			{
+				Type: "git",
+				Name: repository.DefaultTemplateRepositoryName,
+				Url:  repository.DefaultTemplateRepositoryUrl,
+				Ref:  "api-demo",
+			},
+		}, nil
```

Comment:
Hmm, koukám že ten default repozitář se skládá i tady a moc nevím proč, ale je tam rozdíl v `Ref: "main"`:

https://github.com/keboola/keboola-as-code/blob/74776ae05d1fef7b79cd0231a1635b0e71010975/internal/pkg/template/repository/default.go#L13-L20

Reply:
Namiesto tohto miesta, kde je `api-demo` by som to dal do default hodnotu tej flag (alebo do `Dockerfile`, a default hodnota moze byt rovnako `main`).
To druhe miesto co linkujes je pre project `manifest.json`, tj. tam chceme `main`, ale v API mame zatial `api-demo`.